google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts
+Before=shutdown.target reboot.target halt.target
+DefaultDependencies=true
+Wants=local-fs.target network-online.target network.target
```

Comment:
"DefaultDependencies=" should be No
"Wants=" can be removed.

See iptables-store.service for a good example of a service that runs on systemd shutdown.